runtime/src/lib.rs

```diff
@@ -1804,7 +1864,7 @@ impl_runtime_apis! {
 
         fn dispatch_benchmark(
             config: frame_benchmarking::BenchmarkConfig
-        ) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, sp_runtime::RuntimeString> {
+        ) -> Result<Vec<frame_benchmarking::BenchmarkBatch>, alloc::string::String> {
```

Comment:
That is something that maybe we missed before: we can works directly wit `alloc` and `core` instead of using `sp_std` crate to access. If it's true is better to try to fix it around all the codebase.